ui/components/CustomAvatar.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {Badge, Box} from "@mui/material";
+import { styled } from "@mui/material/styles";
+
+
+const StyledBadge = styled(Badge)(({ theme }) => ({
+  backgroundColor : '#44b700',
+    color : '#44b700',
```

Comment:
do we have color palette for this?

Reply:
I just realised that this color doesn't change anything . @Abhishek-kumar09  Other changes done. 
The dco is failing again and again.

Reply:
No problem for DCO rn

Reply:
Getting you covered on DCO here.